CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # master [(unreleased)](https://github.com/whitesmith/rubycritic/compare/v4.0.2...master)
 
+* [BUGFIX] Fix CSS padding-right property spelling (by [@jbampton][])
+* [BUGFIX] Fix HTML5 aria-hidden attribute (by [@jbampton][])
+* [BUGFIX] Fix spelling (by [@jbampton][])
+* [CHANGE] Add HTML img alt attribute to logo (by [@jbampton][])
+* [CHANGE] Remove default type="text/javascript" from HTML5 script tag (by [@jbampton][])
```

Comment:
Many thanks for caring for this project enough to clean the small pieces.

One thing that can be improved here is to keep the changelog small and simple, so that it is easy to glance at and find info. Perhaps all these changes can be packed into less lines? 

Examples:
```
* [BUGFIX] Assorted fixes in HTML report (by [@jbampton][])
* [CHANGE] Improvements to HTML report (by [@jbampton][])
```